dvc/fs/oss.py

```diff
@@ -43,38 +42,3 @@ def fs(self):
 
     def remove(self, path_info):
         self.fs.rm_file(self._with_bucket(path_info))
-
-    def _upload(
-        self, from_file, to_info, name=None, no_progress_bar=False, **kwargs
```

Comment:
@efiop 
Removing these lines will slow down `pull` by 20%, and `push` by about 100%.

Reply:
wait a minute, why would it slow down push/pull? There is no behavioral change

Reply:
@isidentical , Sorry, hadn't seen that the `_upload` and `_download` had been updated. The old `open(fileA, 'w')` version are slower.